manifests/0000_70_dns-operator_02-deployment.yaml

```diff
@@ -36,6 +36,31 @@ spec:
         resources:
           requests:
             cpu: 10m
+      - name: kube-rbac-proxy
+        image: quay.io/openshift/origin-kube-rbac-proxy:latest
+        args:
+        - --logtostderr
+        - -v=8
+        - --secure-listen-address=:9393
+        - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+        - --upstream=http://127.0.0.1:60000/
+        - --tls-cert-file=/etc/tls/private/tls.crt
```

Comment:
Will this image handle automatically reloading a rotated certificate?

Reply:
Yes, kube-rbac-proxy watches for updates (https://github.com/brancz/kube-rbac-proxy/pull/47/commits/ef4527d98d8e3e1ea24c188438b4c416d00bcf08), and I verified manually using the following command:
```console
% oc -n openshift-dns-operator delete secrets/metrics-tls
secret "metrics-tls" deleted
```
Kube-rbac-proxy logs the following:
```console
I1118 17:33:29.709147       1 reloader.go:98] reloading key /etc/tls/private/tls.key certificate /etc/tls/private/tls.crt
```